src/SeleniumLibrary/keywords/alert.py

```diff
@@ -133,6 +133,9 @@ def _wait_alert(self, timeout=None):
         wait = WebDriverWait(self.driver, timeout)
         try:
             return wait.until(EC.alert_is_present())
-        except WebDriverException:
+        except TimeoutException:
             raise AssertionError('Alert not found in %s.'
-                                 % secs_to_timestr(timeout))
+                             % secs_to_timestr(timeout))
+        except WebDriverException as e:
+            raise AssertionError('An exception occurred waiting for alert: %s'
+                                 % str(e))
```

Comment:
`str(e)` fails if the error contains non-ASCII characters on Python 2. Good news is that `str()` cam just be removed and it ought to work fine.

`e` alone perhaps looks a bit odd and could be renamed to `err` at the same time. I think that's used more in SL code base.

Reply:
I've made those changes in commit https://github.com/robotframework/SeleniumLibrary/pull/1501/commits/d61a8030d313bb2f697e07c4334e7992ba9570e3